src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -54,6 +54,9 @@ GcsServer::GcsServer(const ray::gcs::GcsServerConfig &config,
     : config_(config),
       storage_type_(GetStorageType()),
       main_service_(main_service),
+      pubsub_io_context_("pubsub_io_context"),
```

Comment:
InternalPubSubHandler should own it?

Reply:
per offline discussion we keep it in gcs server